validation/src/required.ts

```diff
@@ -1,7 +1,7 @@
 import { ValidationErrors } from 'ngrx-forms';
 
 /**
- * A validation function that requires the value to be non-`null` and non-empty.
+ * A validation function that requires the value to be non-`undefined` and non-empty.
```

Comment:
Please mention both `null` and `undefined`, i.e. "A validation function that requires the value to be non-`null`, non-`undefined`, and non-empty."